src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp

```diff
@@ -275,10 +275,30 @@ void MaEditorNameList::paintEvent(QPaintEvent *) {
     drawAll();
 }
 
+void MaEditorNameList::mouseDoubleClickEvent(QMouseEvent *e) {
+    if (editor->gotoSelectedReadAction->isEnabled()) {
+        editor->gotoSelectedReadAction->trigger();
+        e->ignore();
+        return;
+    }
+    QWidget::mouseDoubleClickEvent(e);
+}
+
 void MaEditorNameList::keyPressEvent(QKeyEvent *e) {
     int key = e->key();
-    bool isShiftPressed = e->modifiers().testFlag(Qt::ShiftModifier);
+    Qt::KeyboardModifiers modifiers = e->modifiers();
+
+    if (modifiers == Qt::NoModifier && (key == Qt::Key_Space || key == Qt::Key_Enter || key == Qt::Key_Return)) {
```

Comment:
You have a `switch`-`case` below, it'd be better if you move it there

Reply:
Fixed.